### ISSUE TITLE
Not a plain file error

### ISSUE THREAD

**Issue by austinbutler** [buo/homebrew-cask-upgrade#21]
Up until today "brew cu" has been working great! But today I get this error, then it stops:

```
(1/31) apache-directory-studio: up to date
(2/31) atom: 1.10.2 -> 1.11.0
(3/31) discord: up to date
(4/31) docker: up to date
Error: File '/Users/abutler/dropbox' is not a plain file
```

What it is complaining about is a folder, not a file. It's where I have my Dropbox folder.

**Comment by iTwenty** [comment]
I am getting the same thing.

**Comment by austinbutler** [comment]
Looks like the underlying issue might be this: https://github.com/caskroom/homebrew-cask/issues/25496

**Comment by miccal** [comment]
[caskroom/homebrew-cask#25611](https://github.com/caskroom/homebrew-cask/issues/25611)

**Comment by jawshooah** [comment]
Fixed in https://github.com/Homebrew/brew/pull/1315.

**Comment by lanox** [comment]
I still seem to be having this issue ?

❯ brew --version
Homebrew 1.0.8
Homebrew/homebrew-core (git revision 6804; last commit 2016-10-21)

❯ brew cask --version
Homebrew-Cask 1.0.8
caskroom/homebrew-cask (git revision 4e613; last commit 2016-10-22)

**Comment by jawshooah** [comment]
@lanox there hasn't been a weekly release yet since the fix was merged in. If you don't want to wait, just `cd "$(brew --repository)" && git checkout master && git pull origin master`

**Comment by lanox** [comment]
@jawshooah thanks.

**Comment by adidalal** [comment]
Should be fixed and merged into stable now (as of 1.0.9)

**Comment by buo** [comment]
Thanks for all your comments guys! Now I will close this issue and mark as resolved.
